crew

```diff
@@ -30,49 +33,11 @@ Usage:
 version 0.4.3
 DOCOPT
 
-ARCH = `uname -m`.strip
-ARCH_LIB = if ARCH == 'x86_64' then 'lib64' else 'lib' end
-
-CREW_PREFIX = '/usr/local'
-CREW_LIB_PREFIX = CREW_PREFIX + '/' + ARCH_LIB
-
-CREW_LIB_PATH = CREW_PREFIX + '/lib/crew/'
-CREW_CONFIG_PATH = CREW_PREFIX + '/etc/crew/'
-CREW_BREW_DIR = CREW_PREFIX + '/tmp/crew/'
-CREW_DEST_DIR = CREW_BREW_DIR + 'dest'
-CREW_DEST_PREFIX = CREW_DEST_DIR + CREW_PREFIX
-CREW_DEST_LIB_PREFIX = CREW_DEST_DIR + CREW_LIB_PREFIX
-
-# Set CREW_NPROC from environment variable or `nproc`
-if ENV["CREW_NPROC"].to_s == ''
-  CREW_NPROC = `nproc`.strip
-else
-  CREW_NPROC = ENV["CREW_NPROC"]
-end
-
-# Set CREW_NOT_COMPRESS from environment variable
-CREW_NOT_COMPRESS = ENV["CREW_NOT_COMPRESS"]
-
-# Set CREW_NOT_STRIP from environment variable
-CREW_NOT_STRIP = ENV["CREW_NOT_STRIP"]
-
-# Set XZ_OPT environment variable for build command.
-# If CREW_XZ_OPT is defined, use it by default.  Use `-7e`, otherwise.
-if ENV["CREW_XZ_OPT"].to_s == ''
-  ENV["XZ_OPT"] = "-7e"
-else
-  ENV["XZ_OPT"] = ENV["CREW_XZ_OPT"]
-end
-
-USER = `whoami`.chomp
-
-# Add lib to LOAD_PATH
-
-$LOAD_PATH.unshift "#{CREW_LIB_PATH}lib"
+# Disallow sudo
+abort "Chromebrew should not be run as root.".lightred if Process.uid == 0
```

Comment:
I'm actually thinking this should go above everything.  We want this to fail as early as possible.

Reply:
Well, it has to be after the string colorization anyway, since it is displayed in red. So, as of now, before the sudo check there are just some require's and the constants definition, is that a problem?

Reply:
I was the one who added the colorization section so I definitely understand.  I was just thinking since you have the `require_relative 'lib/color'` line now, it could go in immediately after that section.

Reply:
Done